pkg/apis/scope/v0alpha1/types.go

```diff
@@ -35,22 +47,22 @@ type ScopeList struct {
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-type ScopeDashboard struct {
+type ScopeDashboardBinding struct {
```

Comment:
as modeled, this would allow multiple bindings to define the same relationship -- i think that is OK, but good to know.

Also are there other things we may want to bind?  If so, should we use a ScopeBinding and include kind in the body?  Something like: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-example

Reply:
> as modeled, this would allow multiple bindings to define the same relationship

I would not mind requireing uniqeness of `scopeid -> dashboardid` relationships. How would that be enforced?

Reply:
Merging this for now regardless :)